sdk/keyvault/azure-security-keyvault-keys/test/macros-build/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required (VERSION 3.13)
+
+project (macros-test LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_executable (
+  macros-build
+  macros_build.cpp
+  macros_build.hpp)
+
+create_per_service_target_build_for_sample(keyvault macros-build)
+
+target_link_libraries(macros-build PRIVATE azure-security-keyvault-keys)
```

Comment:
Is this included anywhere to be built/run in CI, as a test? Should we have this as a gtest test, with an EXPECT for the value?